tests/integration/profiling/rule_based_profilers/test_run_profiler_notebook.py

```diff
@@ -114,3 +114,55 @@ def test_run_data_assistants_notebook(tmp_path):
                 "Files were already deleted by running the optional last cell in the notebook. "
                 "We therefore allow the test to pass"
             )
+
+
+def test_run_self_initializing_expectations_notebook(tmp_path):
+    """
+    What does this test and why?
+    One of the resources we have for self-initializing Expectations is a Jupyter notebook that explains/shows the components in code.
+    This test ensures the codepaths and examples described in the Notebook actually run and pass, nbconvert's
+    `preprocess` function.
+    """
+    base_dir: str = file_relative_path(
+        __file__, "../../../test_fixtures/rule_based_profiler/example_notebooks"
+    )
+    notebook_path: str = os.path.join(base_dir, "SelfInitializingExpectations.ipynb")
+    # temporary output notebook for traceback and debugging
+    output_notebook_path: str = os.path.join(
+        tmp_path, "SelfInitializingExpectations_executed.ipynb"
+    )
+
+    with open(notebook_path) as f:
+        nb = nbformat.read(f, as_version=4)
+
+    ep = ExecutePreprocessor(timeout=60, kernel_name="python3")
+
+    try:
+        ep.preprocess(nb, {"metadata": {"path": base_dir}})
+    except CellExecutionError:
+        msg = 'Error executing the notebook "%s".\n\n' % notebook_path
+        msg += 'See notebook "%s" for the traceback.' % output_notebook_path
+        print(msg)
+        raise
+    finally:
+        with open(output_notebook_path, mode="w", encoding="utf-8") as f:
+            nbformat.write(nb, f)
+        # clean up Expectations directory after running test
+        try:
+            shutil.rmtree(os.path.join(base_dir, "great_expectations/expectations/tmp"))
+            os.remove(
```

Comment:
Love that this is under test!! And that you are looking for issues in each notebook cell. One minor suggestion here is to use [`os.path.exists()`](https://docs.python.org/3/library/os.path.html#os.path.exists) to check for existence of the file before deleting instead of a try/except. Breaking up the check by file has the added bonus of deleting the `new_expectation_suite.json` in case the `.ge_store_backend_id` doesn't exist. You could even do it in a loop over a list of files to check/delete.

Reply:
Thank you for this suggestion. I think this is great... would it be ok to include in a follow-up PR? It's a change that will affect 2 other tests.

Reply:
Glad you like it! No problem on including in a follow-up PR. :)